ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -1742,6 +1740,18 @@ extension SignalProducerProtocol {
 	}
 }
 
+internal final class ReplayValuesLense<Value, Error: Swift.Error> {
```

Comment:
Related benchmark: https://gist.github.com/andersio/f6af7b65a99cccbe77acb5fdbaf67303
Swift 3 beta 4. Compiled with `-O`. Results show that using closure is 20x slower. Since only `ReplayState.values` has to be lensed so far, a hardcoded lense is used.